rts/lra/lra-annotations/src/main/java/org/jboss/narayana/rts/lra/annotation/NestedLRA.java

```diff
@@ -29,38 +29,36 @@
 import java.lang.annotation.Target;
 
 /**
+ * <p>
  * Used on the interface or class. Defines that the container will create
  * a new LRA for each method invocation, regardless of whether or not there is
  * already an LRA associated with the caller. These LRAs will then
  * either be top-level LRAs or nested automatically depending upon the
  * context within which they are created.
- *
+ * <p>
  * When a nested LRA is closed its' compensators are completed but retained. At any time prior to the enclosing LRA
  * being closed or cancelled the nested LRA can be told to compensate (even though it may have already been told
  * to complete).
- *
- * Compatability with the @LRA annotation: if @LRA is not present @Nested is ignored, otherwise the behaviour depends
- * upon the value of the Type attribute:
- *
- * REQUIRED
```

Comment:
How come you aren't documenting the attributes?

Reply:
I'm sorry I don't understand precisely. The change only adds the formatting `<p>` attribute to have better readability. But the text should be the same still.